src/test/java/joelrojastest/Requests.java

```diff
@@ -0,0 +1,39 @@
+package joelrojastest;
+
+import entities.Project;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public class Requests {
+    public static final String TOKEN = "acea0f034eb75c9d8ed7ca11d0f74376f5b7bba8";
+    public static final String URL = "https://api.todoist.com/rest/v1/";
+
+    public static Response createAProject(Project project) {
+        String url = URL + "projects";
+        RequestSpecification requestSpecification = given()
+                .auth().oauth2(TOKEN)
+                .contentType(ContentType.JSON)
+                .body(project)
+                .log().body();
+        return postRequest(requestSpecification, url);
+    }
+
+    public static Response getAProject(String id) {
+        String url = URL + "projects/{id}";
```

Comment:
strings like should be in enums, static variables "projects/{id}", "projects", "id"

Reply:
I fixed it. Thanks